src/components/Drawer.vue

```diff
@@ -81,41 +73,14 @@ export default {
   },
   methods: {
     pushClose(){
-      console.log("abbbbb")
       this.$emit('closeDrawer')
     },
-    push1() {
-      this.drawerflg = false
-      event.preventDefault()
-      this.$SmoothScroll(
-        document.querySelector('#main'),
-        800,
-        'y'
-      )
-    },
-    push2() {
-      this.drawerflg = false
-      event.preventDefault()
-      this.$SmoothScroll(
-        document.querySelector('#aboutSection'),
-        800,
-        'y'
-      )
-    },
-    push3() {
-      this.drawerflg = false
-      event.preventDefault()
-      this.$SmoothScroll(
-        document.querySelector('#skillSection'),
-        800,
-        'y'
-      )
-    },
-    push4() {
-      this.drawerflg = false
+    push(target) {
+      // this.drawerflg = false
+      this.pushClose()
```

Comment:
```suggestion
      this.$emit('closeDrawer');
```
として、pushCloseメソッド消すのがベターかな
pushCloseが他のどこからも呼ばれてないからメソッド定義している意味がない